about.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-OERcA2EqjJCMA+/3y+gxIOqMEjwtxJY7qPCqsdltbNJuaOe923+mo//f6V8Qbsw3" crossorigin="anonymous"></script>
+     <link href="https://fonts.googleapis.com/css2?family=Poppins" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>about us</title>
+</head>
+<body>
+    <section class="container">
+    <div class="navheader">
+        <div class="black">
+          <ul>
+            <li><a href="#"><img src="images/twitterlogo.png" alt="twitterlogo"></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">Mypage</a></li>
+            <li><a href="#">Log out</a></li>
+       </ul>
+        </div>
+    </div>
+
+    <nav>
+        <div class ="navbar1">
+            <img src="images/menu.png" id="menu-icon" alt="humbergur images">
+            <div class="desk-menu" id="deskMenu">
+                <div class="desk-menu-inside">
+                <div class="logo"><img src="images/africalogo.webp" alt="logo images"></div>
+                <ul>
+                  <li ><a class="menu" href="index.html">Home</a></li>
+                  <li ><a class="menu" href="./about.html">About</a></li>
+                  <li ><a class="menu" href="#about.html">Program</a></li>
+                  <li ><a class="menu" href="#contact-me">Join</a></li>
+                  <li ><a class="menu" href="#about-me">Sponser</a></li>
+                  <li ><a class="menu" href="#contact-me">Register</a></li>
+                  <li ><a href="index.html"> <button class="menu-btn"> Presentation</button></a></li>
+                </ul>
+            </div>
+            </div>
+        
+        <div class="mobile-menu" id="mobileMenu">
+            <img id="menu-close" src="images/closeIcon.png" alt="close icon">
+            <ul class="mob-menu">
+              <li class="menu"><a class="moblink-menu" href="./index.html">Home</a></li>
+              <li class="menu"><a class="moblink-menu" href="#about.html">Program</a></li>
+              <li class="menu"><a class="moblink-menu" href="#contact-me">Join</a></li>
+              <li class="menu"><a class="moblink-menu" href="#about-me">Sponser</a></li>
+              <li class="menu"><a class="moblink-menu" href="#contact-me">Register</a></li>
+            </ul>
+          </div>
+         </div>
+      </nav>
```

Comment:
- Like the previous reviewer suggested and as I pointed in in my review for the index page, please avoid repeating yourself